src/fetch.js

```diff
@@ -1,5 +1,21 @@
-let fetch = {
-  
+let apiFetch = {
+  getData(url) {
+    return fetch(url)
+      .then(res => res.json())
+      .catch(err => console.log(err))
+  },
+
+  postData(url, tripInfo) {
+    const options = {
+      method: "POST",
+      headers: { 'Content-type': 'application/json'},
+      body: JSON.stringify(tripInfo)
+    }
+    return fetch(url, options)
+      .then(res => res.json())
+      .catch(err => console.log(err))
+  }
+
```

Comment:
Nice work incorportaing the catch's in here, this always seems to be thing I forget! Excellent work on this Chris!